src/lib/Libattr/attr_fn_entlim.c

```diff
@@ -64,9 +64,9 @@
  * 	This file contains functions for manipulating attributes of type "entlim"
  * 	entity limits for Finer Granularity Control (FGC)
  * 	This layer is to somewhat isolate the entlim concept from the specific
- * 	implementation (avl tree).
+ * 	implementation (index tree).
```

Comment:
index tree does not make sense - I think we can remove the sentence from line 66